### PR TITLE
first draft dutch translation, needs review

### PR DESCRIPTION
**Pull request by jgroenen**
Needs a review first, before pushing to live. Anyone available to do this?

### CODE REVIEW

File: site/nl.njk [hunk 1]
```diff
@@ -0,0 +1,128 @@
+---
+layout: default
+meta_title: Een beweging om de COVID-19 Pandemic te stoppen
```
Comment (aucuparia) [inline site/nl.njk]:
```suggestion
meta_title: Een beweging om de COVID-19 pandemie te stoppen
```

File: site/nl.njk [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+        <h1 class="page-title">{{ title }}</h1>
+        <h2 class="page-subtitle">{{ meta_title }}</h2>
+
+        <div class="content">
+
+            <p class="text-xl">
+                Onze overheden lopen achter bij het voorkomen van de verspreiding van
+                <a href="https://en.wikipedia.org/wiki/Severe_acute_respiratory_syndrome_coronavirus_2">SARS-CoV-2</a>
+                en het bedwingen van de <a href="https://en.wikipedia.org/wiki/Coronavirus_disease_2019">COVID-19 pandemie</a>.
+                Langzame reacties, zoethoudertjes, en de behoefte om de economie te stabiliseren weerhoudt ze ervan de maatregelen te nemen die nodig zijn om miljoenen mensen te beschermen tegen deze ziekte. Het is tijd dat wij, de inwoners van deze planeet,
```
Comment (aucuparia) [inline site/nl.njk]:
```suggestion
                Langzame reacties, zoethoudertjes, en de behoefte om de economie te stabiliseren weerhoudt ze ervan de maatregelen te nemen die nodig zijn om miljoenen mensen te beschermen tegen deze ziekte. Het is tijd dat wij, burgers van deze planeet,
```

File: site/nl.njk [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+                direct zelf actie nemen en onze bijdrage leveren aan het bestrijden van COVID-19.
+                <br /><br />
+                Recht voor zijn raap gezegd: <strong>Blijf De Fuck Thuis!</strong>
+            </p>
+
+            <h2 class="text-3xl">Manifest voor Zelfisolatie</h2>
+
+            <p>
+                Omdat er nog geen wetenschappelijk bewezen behandeling is, en een vaccin ook nog zeker een jaar op zich laat wachten, kunnen we de coronaviruspandemie alleen effectief tegengaan door het virus minder kans te geven zich te verspreiden. De volgende lijst van acties, gerangschikt van eenvoudig te implementeren naar meest effectief in de strijd tegen de pandemie, kan dienen als een losse set van richtlijnen voor mensen die willen bijdragen aan deze beweging en zelf iets door, zolang bestuurders reactief blijven.
```
Comment (aucuparia) [inline site/nl.njk]:
```suggestion
                Omdat er nog geen wetenschappelijk bewezen behandeling is, en een vaccin ook nog zeker een jaar op zich laat wachten, kunnen we de coronaviruspandemie alleen effectief tegengaan door het virus minder kans te geven zich te verspreiden. De volgende lijst van acties, gerangschikt van eenvoudig te implementeren naar meest effectief in de strijd tegen de pandemie, kan dienen als een losse set van richtlijnen voor mensen die willen bijdragen aan deze beweging en in actie willen komen waar onze leiders tekort schieten.
```

File: site/nl.njk [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+            </p>
+
+            <ol class="text-xl mb-12 list-decimal pl-12">
+                <li class="pb-2">Geen paniek, maar doe voorzichtig.</li>
```
Comment (aucuparia) [inline site/nl.njk]:
```suggestion
                <li class="pb-2">Raak niet in paniek, maar wees wel voorzichtig.</li>
```

File: site/nl.njk [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+                <li class="pb-2">Was je handen regelmatig, en hoest en nies in je elleboogholte of zakdoek.</li>
```
Comment (aucuparia) [inline site/nl.njk]:
```suggestion
                <li class="pb-2">Was je handen vaak, en hoest en nies in je elleboogholte of zakdoek.</li>
```

File: site/nl.njk [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+                <li class="pb-2">Probeer je hoofd zo min mogelijk aan te raken, vooral je mond, neus, en ogen.</li>
```
Comment (aucuparia) [inline site/nl.njk:38]:
```suggestion
                <li class="pb-2">Probeer je hoofd zo min mogelijk aan te raken. Vermijd vooral je mond, neus, en ogen.</li>
```

File: site/nl.njk [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+                <li class="pb-2">Houd afstand tot anderen (social distance), geen omhelzingen en kusjes, geef elkaar geen hand en geen high fives. Als het echt nodig is, gebruik dan een veiliger alternatief.</li>
```
Comment (aucuparia) [inline site/nl.njk:39]:
```suggestion
                <li class="pb-2">Houd afstand tot anderen (social distancing), geen omhelzingen en kusjes, geef elkaar geen hand en geen high fives. Als het echt nodig is, gebruik dan een veiliger alternatief.</li>
```

File: site/nl.njk [hunk 8]
```diff
@@ -0,0 +1,128 @@
...
+                <li class="pb-2">Ga niet naar concerten, toneel, sportwedstrijden, en andere grootschalige bijeenkomsten.</li>
+                <li class="pb-2">Ga niet naar musea, tentoonstellingen, bioscopen, café's en clubs, en andere plekken van vermaak.</li>
```
Comment (aucuparia) [inline site/nl.njk:41]:
```suggestion
                <li class="pb-2">Ga niet naar musea, tentoonstellingen, bioscopen, cafés en clubs, en andere plekken van vermaak.</li>
```

File: site/nl.njk [hunk 9]
```diff
@@ -0,0 +1,128 @@
...
+                <li class="pb-2">Blijf weg bij sociale bijeenkomsten en activiteiten, zoals verenigingsactiviteiten, religieuze diensten, en privéfeestjes.</li>
+                <li class="pb-2">Beperk het reizen tot een minimum. Reis geen lange afstanden als het niet strikt noodzakelijk is.</li>
+                <li class="pb-2">Ga niet met het openbaar vervoer als het niet absoluut nodig is.</li>
```
Comment (aucuparia) [inline site/nl.njk:44]:
```suggestion
                <li class="pb-2">Ga niet met het openbaar vervoer als het niet absoluut onvermijdelijk is.</li>
```

File: site/nl.njk [hunk 10]
```diff
@@ -0,0 +1,128 @@
...
+                <li class="pb-2">Als je vanuit huis kunt werken, doe dat dan. Spoor je werkgever indien nodig aan om dit toe te staan.</li>
+                <li class="pb-2">Vervang zo veel mogelijk sociale interacties door alternatieven op afstand zoals telefoneren en video chat.</li>
```
Comment (aucuparia) [inline site/nl.njk:46]:
```suggestion
                <li class="pb-2">Vervang zo veel mogelijk sociale interacties door alternatieven op afstand zoals bellen of een video-oproep.</li>
```

File: site/nl.njk [hunk 11]
```diff
@@ -0,0 +1,128 @@
...
+                <li class="pb-2">Als het niet nodig is om naar buiten te gaan, blijf dan binnen.</li>
```
Comment (aucuparia) [inline site/nl.njk:47]:
```suggestion
                <li class="pb-2">Als het niet noodzakelijk is om naar buiten te gaan, blijf dan binnen.</li>
```

File: site/nl.njk [hunk 12]
```diff
@@ -0,0 +1,128 @@
...
+            </ol>
+
+            <p class="text-xl">
+                Onthoud dat er geen goed of fout is in de mate waarin je dit opvolgt. Doe alleen de dingen waar je je goed bij voelt, en die je kostwinning niet in gevaar brengen. <strong>Laat dit niet leiden tot je ontslag!</strong>
```
Comment (aucuparia) [inline site/nl.njk:51]:
```suggestion
                Onthoud dat er geen goed of fout is in de mate waarin je dit opvolgt. Doe alleen de dingen waar je je goed bij voelt, en die je kostwinning niet in gevaar brengen. <strong>Zeg hier je werk niet voor op!</strong>
```

File: site/nl.njk [hunk 13]
```diff
@@ -0,0 +1,128 @@
...
+                Houd echter wel in gedachten dat alle beetjes helpen.
+            </p>
+
+            <h2 class="text-3xl">Waarom is dit zo belangrijk?</h2>
+
+            <p>
+                SARS-CoV-2 is een zeer besmettelijk en voor voor een deel van de mensen dodelijk virus dat leidt tot de ademhalingsziekte COVID-19.
```
Comment (aucuparia) [inline site/nl.njk:58]:
```suggestion
                SARS-CoV-2 is een zeer besmettelijk en potentieel dodelijk virus dat leidt tot de ziekte van de luchtwegen genaamd COVID-19.
```

File: site/nl.njk [hunk 14]
```diff
@@ -0,0 +1,128 @@
...
+                Je kent het misschien onder een andere naam, als 2019-nCoV, het nieuwe coronavirus, het Wuhan coronavirus, de Chineese- of Wuhangriep, of simpelweg het coronavirus.
```
Comment (aucuparia) [inline site/nl.njk:59]:
```suggestion
                Je kent het misschien onder een andere naam, bijvoorbeeld 2019-nCoV, het nieuwe coronavirus, het Wuhan coronavirus, de Chineese- of Wuhangriep, of simpelweg het coronavirus.
```

File: site/nl.njk [hunk 15]
```diff
@@ -0,0 +1,128 @@
...
+                Al deze namen refereren naar hetzelfde virus, en dat is het virus dat deze beweging probeert tegen te houden.
+            </p>
+
+            <p>
+                Gedurende de afgelopen maanden en weken werden het virus en de bijbehorende ziekte keer op keer vergeleken met "gewone" seizoensgriep, waarschijnlijk omdat het qua symptomen en ernst op elkaar lijkt. Echter, met de huidige kennis over dit nieuwe virus, moet de bedreiging die het vormt voor de samenleving niet zo eenvoudig opzij worden geschoven.
```
Comment (aucuparia) [inline site/nl.njk:64]:
```suggestion
                Gedurende de afgelopen maanden en weken werden het virus en de bijbehorende ziekte keer op keer vergeleken met "gewone" seizoensgriep. Door overeenkomsten in symptomen en schijnbaar vergelijkbare ernst, is dit voor veel mensen een logische vergelijking. Echter, met de huidige kennis over dit nieuwe virus, moet de bedreiging die het vormt voor de samenleving niet gebagatelliseerd worden.
```

File: site/nl.njk [hunk 16]
```diff
@@ -0,0 +1,128 @@
...
+            </p>
+
+            <ol class="text-xl mb-12 list-decimal pl-12 font-bold">
+                <li class="pb-2">
+                    <h3>Besmettelijker dan de griep</h3>
+                    <p class="font-normal">
+                        Met een geschatte R0 van tussen de 1.4 &ndash; 6.49 en een gemiddelde schatting van 3.28<sup><a class="source-link" href="https://academic.oup.com/jtm/advance-article/doi/10.1093/jtm/taaa021/5735319" target="_blank">[1]</a></sup> is
```
Comment (aucuparia) [inline site/nl.njk:71]:
```suggestion
                        Met een geschatte R0 van tussen de 1.4 &ndash; 6.49 en een geschat gemiddelde van 3.28<sup><a class="source-link" href="https://academic.oup.com/jtm/advance-article/doi/10.1093/jtm/taaa021/5735319" target="_blank">[1]</a></sup> is
```

File: site/nl.njk [hunk 17]
```diff
@@ -0,0 +1,128 @@
...
+                        SARS-CoV-2 veel besmettelijker en verspreid het zich een stuk sneller dan de seizoensgriep, die een mediane R0 van 1.28<sup><a class="source-link" href="https://www.ncbi.nlm.nih.gov/pubmed/25186370" target="_blank">[2]</a></sup> heeft.
+                    <p>
+                </li>
+                <li class="pb-2">
+                    <h3>Dodelijker dan de griep</h3>
+                    <p class="font-normal">
+                        De zogenaamde case fatality rate (CFR) van SARS-CoV-2 wordt geschat op ongeveer 2%<sup><a class="source-link" href="https://www.ncbi.nlm.nih.gov/pubmed/32146445" target="_blank">[3]</a></sup>. Dat wil zeggen dat een geschatte 2% van alle mensen die worden besmet met COVID-19 daaraan zullen bezwijken. Ter vergelijking, de CFR van de seizoensgriep wordt geschat op ongeveer 0.1%<sup><a class="source-link" href="https://www.cdc.gov/flu/about/burden/2018-2019.html" target="_blank">[4]</a></sup>, en daarmee is SARS-CoV-2 ongeveer 20 keer zo dodelijk.
```
Comment (aucuparia) [inline site/nl.njk:78]:
```suggestion
                        De zogenaamde case fatality rate (CFR) van SARS-CoV-2 wordt geschat op ongeveer 2%<sup><a class="source-link" href="https://www.ncbi.nlm.nih.gov/pubmed/32146445" target="_blank">[3]</a></sup>. Dat wil zeggen dat naar schatting 2% van de mensen die de diagnose COVID-19 krijgen hier helaas aan zal overlijden. Ter vergelijking, de CFR van de seizoensgriep wordt geschat op ongeveer 0.1%<sup><a class="source-link" href="https://www.cdc.gov/flu/about/burden/2018-2019.html" target="_blank">[4]</a></sup>. Daarmee is SARS-CoV-2 ongeveer 20 keer zo dodelijk als de griep.
```

File: site/nl.njk [hunk 18]
```diff
@@ -0,0 +1,128 @@
...
+                    <p>
+                </li>
+                <li class="pb-2">
+                    <h3>Kans op ziekenhuisopname</h3>
```
Comment (aucuparia) [inline site/nl.njk:82]:
```suggestion
                    <h3>Kans op ernstige ziekteverschijnselen</h3>
```

File: site/nl.njk [hunk 19]
```diff
@@ -0,0 +1,128 @@
...
+                    <p class="font-normal">
+                        Een geschatte 15 &ndash; 20% van alle geïnfecteerde personen krijgen ernstige symptomen waarvoor ziekenhuisopname nodig is<sup><a class="source-link" href="https://www.ncbi.nlm.nih.gov/pubmed/32064853" target="_blank">[5]</a></sup>, waaronder longontsteking met kortademigheid, en een verlaagd zuurstofgehalte in het bloed.
```
Comment (aucuparia) [inline site/nl.njk:84]:
```suggestion
                        Een geschatte 15 &ndash; 20% van alle geïnfecteerde personen krijgen ernstige symptomen die medische hulp vereisen<sup><a class="source-link" href="https://www.ncbi.nlm.nih.gov/pubmed/32064853" target="_blank">[5]</a></sup>, zoals longontsteking met kortademigheid, en een verlaagd zuurstofgehalte in het bloed.
```

File: site/nl.njk [hunk 20]
```diff
@@ -0,0 +1,128 @@
...
+                    </p>
+                </li>
+                <li class="pb-2">
+                    <h3>Geen Behandeling, Geen Vaccin, Geen Immuniteit</h3>
```
Comment (aucuparia) [inline site/nl.njk:88]:
```suggestion
                    <h3>Geen behandeling, geen vaccin, geen immuniteit</h3>
```

File: site/nl.njk [hunk 21]
```diff
@@ -0,0 +1,128 @@
...
+                    <p class="font-normal">
+                        Omdat SARS-CoV-2 pas recent de kop heeft opgestoken, is er nog geen goed bestudeerde behandeling voor COVID-19, en is er dus meer onderzoek nodig, voordat we mensen die geïnfecteerd zijn efficient kunnen behandelen.
```
Comment (aucuparia) [inline site/nl.njk:90]:
```suggestion
                        Omdat SARS-CoV-2 net nieuw is, is er nog geen goed onderzochte behandeling voor COVID-19, en is er dus meer onderzoek nodig voordat we mensen die geïnfecteerd zijn efficiënt kunnen behandelen.
```

File: site/nl.njk [hunk 22]
```diff
@@ -0,0 +1,128 @@
...
+                        Om dezelfde reden is er nog geen vaccin voor SARS-CoV-2 yet<sup><a class="source-link" href="https://www.cdc.gov/coronavirus/2019-ncov/about/prevention-treatment.html" target="_blank">[6]</a></sup> en de ontwikkeling daarvan zal nog zeker een tijd duren. Omdat er nog geen vaccin is, en er nog geen immuniteit is opgebouwd, loopt iedereen risico op besmetting.
```
Comment (aucuparia) [inline site/nl.njk:91]:
```suggestion
                        Om dezelfde reden is er nog geen vaccin voor SARS-CoV-2 yet<sup><a class="source-link" href="https://www.cdc.gov/coronavirus/2019-ncov/about/prevention-treatment.html" target="_blank">[6]</a></sup> en zal de ontwikkeling daarvan nog enige tijd duren. Zonder vaccin en zonder wijd verspreide weerstand tegen dit nieuwe virus is iedereen vatbaar voor besmetting.
```

File: site/nl.njk [hunk 23]
```diff
@@ -0,0 +1,128 @@
...
+                        Alhoewel de meeste besmette personen milde symptomen zullen ervaren, zal dit gebrek aan grootschalige immuniteit leiden tot ernstige ziekte bij een significante groep personen met verhoogd risico.
```
Comment (aucuparia) [inline site/nl.njk:92]:
```suggestion
                        Alhoewel de meeste besmette personen milde symptomen zullen ervaren, kan dit gebrek aan groepsimmuniteit ertoe leiden dat een grote groep kwetsbare personen ernstig ziek zal worden.
```

File: site/nl.njk [hunk 24]
```diff
@@ -0,0 +1,128 @@
...
+                    </p>
+                </li>
+                <li class="pb-2">
+                    <h3>Exponentiele Groei</h3>
```
Comment (aucuparia) [inline site/nl.njk:96]:
```suggestion
                    <h3>Exponentiële groei</h3>
```

File: site/nl.njk [hunk 25]
```diff
@@ -0,0 +1,128 @@
...
+                    <p class="font-normal">
+                        Vanwege het ontbreken van immuniteit tegen dit nieuwe virus, is de gehele bevolking besmetbaar. En ondanks dat exponentiële groei in het begin altijd langzaam lijkt te gaan, leidt het tot onvoorstelbaar hoge aantallen in een kort tijdsbestek. Gezien het aantal besmette personen momenteel iedere paar dagen verdubbeld<sup><a class="source-link" href="https://www.who.int/emergencies/diseases/novel-coronavirus-2019/situation-reports/" target="_blank">[7]</a></sup>, kan dit onze medische systemen overvloeien, wat kan leiden tot een hoger aantal doden, als mensen niet de behandeling krijgen die nodig is.
```
Comment (aucuparia) [inline site/nl.njk]:
```suggestion
                        Doordat er geen immuniteit is voor dit nieuwe virus, is de gehele menselijke bevolking besmetbaar. En ondanks dat exponentiële groei in het begin altijd langzaam lijkt te gaan, kan het tot onvoorstelbaar hoge aantallen leiden in een zeer korte tijd. Gezien het aantal besmette personen momenteel iedere paar dagen verdubbeld<sup><a class="source-link" href="https://www.who.int/emergencies/diseases/novel-coronavirus-2019/situation-reports/" target="_blank">[7]</a></sup>, kan het medisch systeem overbelast raken, waardoor er meer sterfgevallen zullen zijn omdat mensen niet de zorg krijgen die zij nodig hebben.
```

File: site/nl.njk [hunk 26]
```diff
@@ -0,0 +1,128 @@
...
+                    </p>
+                </li>
+            </ol>
+
+            <p class="text-2xl">
+                Door je te voegen bij de beweging en daarmee de kans op nieuwe infecties te verkleinen, bescherm je niet alleen jezelf, maar help je ook de verspreiding beperken voor anderen, vooral voor de mensen die een hoger risico hebben op zeer ernstige gevolgen van dit virus.<br /><br />
```
Comment (aucuparia) [inline site/nl.njk]:
```suggestion
                Door je te voegen bij de beweging en daarmee de kans op nieuwe infecties te verkleinen, bescherm je niet alleen jezelf, maar help je ook de verspreiding beperken voor anderen, vooral voor de mensen die een hoger risico hebben op ernstige gevolgen door dit virus.<br /><br />
```

File: site/nl.njk [hunk 27]
```diff
@@ -0,0 +1,128 @@
...
+
+                <strong>Als je hierom geeft, deel dit dan alsjeblieft op social media, via e-mail, of vertel vrienden en familie er over. En alsjeblief, blijf nou gewoon de fuck thuis! #StayTheFuckHome.</strong>
```
Comment (aucuparia) [inline site/nl.njk]:
```suggestion
                <strong>Als je dit belangrijk vind, deel dit dan op social media, via e-mail, of vertel erover aan vrienden en familie. En alsjeblieft, blijf gewoon de fuck thuis! #StayTheFuckHome.</strong>
```

File: site/nl.njk [hunk 28]
```diff
@@ -0,0 +1,128 @@
...
+            </p>
+
+            <div class="flex justify-center pb-12">
+                <a href="https://twitter.com/share?ref_src=twsrc%5Etfw" class="twitter-share-button" data-size="large" data-text="Ik neem mijn verantwoordelijkheid in het tegengaan van #COVID19. Ik vraag iedereen om hetzelfde te doen!" data-url="https://staythefuckhome.com/nl/" data-hashtags="StayTheFuckHome" data-show-count="false">Tweet</a><script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
```
Comment (aucuparia) [inline site/nl.njk]:
```suggestion
                <a href="https://twitter.com/share?ref_src=twsrc%5Etfw" class="twitter-share-button" data-size="large" data-text="Ik neem mijn verantwoordelijkheid in het tegengaan van #COVID19. Ik verzoek je dringend om hetzelfde te doen!" data-url="https://staythefuckhome.com/nl/" data-hashtags="StayTheFuckHome" data-show-count="false">Tweet</a><script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
```

File: site/nl.njk [hunk 29]
```diff
@@ -0,0 +1,128 @@
...
+                <iframe class="ml-6" src="https://www.facebook.com/plugins/share_button.php?href=https%3A%2F%2Fstaythefuckhome.com&layout=button_count&size=large&appId=764658637392408&width=102&height=28" width="102" height="28" style="border:none;overflow:hidden" scrolling="no" frameborder="0" allowTransparency="true" allow="encrypted-media"></iframe>
+            </div>
+
+            <p class="text-sm text-gray-500 pt-12 max-w-4xl mx-auto">
+                <strong>Disclaimer:</strong> De inhoud van deze website is geen medisch advies. De informatie hierboven is bedoeld als een losse set leefregels voor bezorgde mensen gebaseerd op best practices en anekdotisch bewijs om zichzelf en hun geliefden te beschermen. Er is geen garantie of wetenschappelijk bewijs dat het volgen van deze leefregels zal leiden tot een vermindering van nieuwe infecties of het stoppen van de COVID-19 epidemie. Er is ook geen garantie dat het volgen van deze regels het risico op besmetting met SARS-CoV-2 of andere virussen of bacterien verminderd.
```
Comment (aucuparia) [inline site/nl.njk]:
```suggestion
                <strong>Disclaimer:</strong> De inhoud van deze website is geen medisch advies. De informatie hierboven is bedoeld als een losse richtlijn voor bezorgde mensen gebaseerd op best practices en anekdotisch bewijs om zichzelf en hun geliefden te beschermen. Er is geen garantie of wetenschappelijk bewijs dat het volgen van deze richtlijn zal leiden tot een vermindering van nieuwe infecties of het stoppen van de COVID-19 epidemie. Er is ook geen garantie dat het volgen van deze regels het risico op besmetting met SARS-CoV-2 of andere virussen of bacterien verminderd.
```

**Review by aucuparia — APPROVED** [review]

**Review by flore2003 — APPROVED** [review]

**Comment by flore2003** [comment]
Thanks you so much!

**Comment by ahunts** [comment]
@jgroenen and @aucuparia would you like to go on our attribution page? If so, could I have the name you wish to use?

**Comment by jgroenen** [comment]
No thank you

**Pull request merged**